modules/prebidServerBidAdapter/index.js

```diff
@@ -483,8 +483,25 @@ const OPEN_RTB_PROTOCOL = {
       tmax: _s2sConfig.timeout,
       imp: imps,
       test: getConfig('debug') ? 1 : 0,
+      // ext.prebid.targeting.includewinners is always true for openrtb endpoints
+      ext: {
+        prebid: {
+          targeting: {
+            includewinners: true
+          }
+        }
+      }
     };
 
+    // s2sConfig video.ext.prebid is passed through openrtb to PBS
+    if (utils.deepAccess(_s2sConfig, 'video.ext.prebid')) {
+      request.video = {
```

Comment:
I think that this is supposed to be top level not a video object in the request..

So it would be:

```javascript
request.ext.prebid = utils.deepAccess(_s2sConfig, 'video.ext.prebid')
```

So since we set the includeWinners above thing, probably best to do this:

```javascript
Object.assign(request.ext.prebid, utils.deepAccess(_s2sConfig, 'video.ext.prebid'));
```

I think....

@bretg Can you confirm this?